build/build_zss.sh

```diff
@@ -62,7 +70,7 @@ c89 \
   -DAPF_AUTHORIZED=0 \
   -Wc,dll,expo,langlvl\(extc99\),gonum,goff,hgpr,roconst,ASM,asmlib\('CEE.SCEEMAC','SYS1.MACLIB','SYS1.MODGEN'\) \
   -Wc,agg,exp,list\(\),so\(\),off,xref \
-  -Wl,ac=1 \
+  -Wl,ac=1,dll \
```

Comment:
can you give a short description about the difference here? I see it is a supported flag but I am curious about what was wrong when we only had dll for -Wc, but not -Wl

Reply:
According to the [manual](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.cbcux01/oebind3.htm), the dll compiler options tell the compiler that some symbols will come from a dll, the dll linker option tells the linker to produce a side file and a proper dll.

I've just tested and the build works fine without the compiler dll option. However, the build will produce a warning if there is no linker dll option. I think it's because, we still have the export all option, which makes c89 create a binary with exported symbols, but a missing dll option causes the linker not to allocate the SYSDEFSD (side-deck DD) and produce a warning message.
```
When you bind code with exported symbols, you should specify the DLL
binder option (-W l,dll).
```

Reply:
One of the things this does is generate the side file with every single function. We don't really want that since it is a form of API documentation and we only want a subset of those functions to be externally used in case we change something. So, the .x file is going to be discarded anyway. This PR was an attempt to make an official one: https://github.com/zowe/zss/pull/30
So, is this still better even if we do not use the .x file that gets generated?

Reply:
> So, is this still better even if we do not use the .x file that gets generated?
This is just to make the linker stop producing that warning message. That PR is still relevant. We need to select a subset of the functions we want to export. That is, right now it's "export all" later it'll be specific pragmas, but we'll have to use `-Wl,dll` in either case.

Reply:
That's acceptable to me. I don't know of any way to stop generating a side file with -qdll